src/index.html

```diff
@@ -15,8 +15,95 @@
       rel="stylesheet"
       href="./style.css"
     />
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,100;0,300;0,400;0,500;0,700;0,900;1,100;1,300;1,400;1,500;1,700;1,900&display=swap"
+      rel="stylesheet"
+    />
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a
+        href="/"
+        class="logo"
+      >
+        <img
+          src="/src/images/logo.png"
+          alt="MOYO logo"
+        />
+      </a>
+      <nav>
+        <ul class="nav-list">
+          <li>
```

Comment:
Add class for li everywhere and use this class in css
```suggestion
          <li class="nav__item">
```